src/apps/opening-hours-editor/ConfirmEditRepeatedOpeningHour.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { ChangeEvent, FC, useState } from "react";
+import { useText } from "../../core/utils/text";
+
+enum OptionValue {
+  THIS = "this",
+  All = "all"
```

Comment:
Please streamline the casing here.

Based on a quick look around the codebase it does not seem as if we uppercase enums elsewhere so I suggest changing `THIS` to `This`.